spec/e2e/e2e_helper.rb

```diff
@@ -198,3 +198,12 @@ def bootstrap_flask
   end
   on hosts, 'python3 -m pip install flask'
 end
+
+def dl_ctr_logs
+  dir = 'tmp/e2e_ctr/'.tap { |d| `mkdir -p #{d}` }
+
+  hosts.each do |host|
+    host_dir = "#{dir}/#{host}".tap { |d| `mkdir -p #{d}` }
+    scp_from(host, '/var/log/ondemand-nginx/error.log', host_dir)
```

Comment:
Also be good to get Apache logs, would just need a helper function to determine the appropriate log path and  maybe update ood_portal.yml with desired log name/path so it's more deterministic.